historical_data/raw_data_csv

```diff
@@ -0,0 +1,13 @@
+incoming_amt,incoming_expiry,incoming_add_ts,incoming_remove_ts,outgoing_amt,outgoing_expiry,outgoing_add_ts,outgoing_remove_ts,forwarding_node,forwarding_alias,chan_in,chan_out
```

Comment:
Going to remove this do that docker build will fail if you forget to add your own data, I see future footgunz where we're just copying in this boilerplate data and thinking we're ok.